Dynatrace/dynatrace-operator
# Adds pod annotation to disable mutation

# Description

Adds new pod annotation that will stop the webhook mutation, no matter what else is configured 

## How can this be tested?
Add `dynatrace.com/inject: false` annotation to a sample app, and check that they don't get injected with anything


## Checklist
